pytest_mypy_plugins/configs.py

```diff
@@ -54,6 +55,19 @@ def join_toml_configs(
     with mypy_config_file_path.open("w") as f:
         # We don't want the whole config file, because it can contain
         # other sections like `[tool.isort]`, we only need `[tool.mypy]` part.
-        f.write(f"{_TOML_TABLE_NAME}\n")
-        f.write(dedent(toml_config["tool"]["mypy"].as_string()))  # type: ignore[index]
+        tool_mypy = toml_config["tool"]["mypy"]
+
+        # construct toml output
+        min_toml = tomlkit.document()
+        min_tool = tomlkit.table(is_super_table=True)
+        min_toml.append("tool", min_tool)
+        min_tool.append("mypy", tool_mypy)
+
+        # strip leading comments
+        for i, item in enumerate(min_toml.body):
+            if not isinstance(item, (Comment, Whitespace)):
```

Comment:
Let's add a leading comment to test that line 👍

Reply:
All three pyproject.toml test files have leading comments. I had to add this to ensure the tests for pyproject1.toml and pyproject2.toml pass:

https://github.com/typeddjango/pytest-mypy-plugins/blob/a92e6bcf5a7c2057498df93ea3105b4b11886452/pytest_mypy_plugins/tests/test_configs/pyproject1.toml#L1-L2

https://github.com/typeddjango/pytest-mypy-plugins/blob/a92e6bcf5a7c2057498df93ea3105b4b11886452/pytest_mypy_plugins/tests/test_configs/pyproject2.toml#L1

Reply:
Nevermind.
My local installation was broken. This code path was not required and never used.

I removed it.